### DISCUSSION TITLE
using with nftables, firewalld

### DISCUSSION THREAD

**Discussion by dkebler** [NHAS/wag#113] [Q&A]
My router already has firewalld, nftables installed.   I won't go back to using iptables.   Can I install wag and manage any firewall setting myself which I already do with just plain wireguard.  I am only looking to add MFA to my road warrior wireguard server.

**Comment by dkebler** [comment]
hmm.  I use arch for wondering if I load `iptables-nft`  then if wag tries to create an iptable rule a corresponding nftable rule is created? 

https://wiki.archlinux.org/title/Nftables

**Comment by NHAS** [comment]
Wag does not support anything other than IPtables. This is noted in the readme
